tf-build-tools/tf-maven-plugin/src/main/java/com/deliveredtechnologies/maven/terraform/TerraformPackage.java

```diff
@@ -100,6 +103,13 @@ public String execute(Properties properties) throws TerraformException {
               : TerraformUtils.getDefaultTerraformRootModuleDir());
       logger.debug(String.format("tfRootPath is %1$s", tfRootPath.toAbsolutePath().toString()));
 
+      List<Path> tfVarFilePaths = StringUtils.isEmpty(tfVarFiles)
```

Comment:
This method is starting to get a little large. Maybe we should break it down and encapsulate some of this logic into other methods.